pypeit/flatfield.py

```diff
@@ -215,8 +215,10 @@ class FlatField(object):
     For the primary methods, see :func:`run`.
 
     Args:
-        rawflatimg (:class:`pypeit.images.pypeitimage.PypeItImage`):
+        rawpixflatimg (:class:`pypeit.images.pypeitimage.PypeItImage`):
```

Comment:
Can this be `None` if an illumflat is provided?

Reply:
As above, if `illumflat` is None, the pixel and illumination correction is done on `pixflat`. Otherwise, the illumflat is used for the illumination and pixflat is used for the pixel flat.